Lambda-School-Labs/mydish-fe
# Testing/release canvas 1

- adds snapshot tests for CreateRecipeForm, ImageUploadModal, and RecipeImage
- fixes 7 of the failing Redux store and action creator tests

**To test:**
1. Check out branch.
2. Run `npm test`.
3. Check that test results look like those below. We still have 10 failing tests. We'll see if we can solve the mystery of their failing.
![Screen Shot 2020-03-13 at 12 17 33 PM](https://user-images.githubusercontent.com/50860480/76653111-4ab82980-6525-11ea-9cb5-2ca7e2056e9a.jpg)

*Note:* If you change anything in the UI for a particular component and then run tests again, the snapshot test for that component will fail. Run `jest --updateSnapshot` first, then run `npm test` again. (If you don't have the jest CLI installed, do so by running `npm install jest --global`.)
